download/index.md

```diff
@@ -2,17 +2,17 @@
 layout: download
 title: Download
```

Comment:
I suggest adding:

~~~ yaml
redirect_from:
  - /download/scala2.html
  - /download/scala3.html
~~~

To not break the possible existing links that point to these URLs.

Reply:
thanks!